roles/bridge/vars/main.yml

```diff
@@ -2,10 +2,17 @@
 # maintains file name, binary names, and anything which is not a directory or path
 
 # config file names
-bridge_config_file_name: config.yaml
-bridge_authority_key_file_name: authority.key
-bridge_client_key_file_name: client.key
+bridge_config_file_name: config.toml
+bridge_node_key_oaz_file_name: OAZHALLLMV4Q
+bridge_node_key_nj3_file_name: NJ3XILLTMVRXEZLUFZVHO5A
 
 # binary names
-bridge_service_default_binary_name: sui-bridge
-bridge_cli_default_binary_name: sui-bridge-cli
+bridge_default_binary_name: celestia
+bridge_cel_key_binary_name: cel-key
+
+bridge_core_ip:
```

Comment:
move this to group var too

```ansible
bridge_core_ip:
mainnet: consensus.lunaroasis.net
testnet: rpc-mocha.pops.one

bridge_core_rpc_port: "26657"
bridge_core_grpc_port: "9090"
```


these ports are for the rpc we are using, create a structure like this maybe

```ansible
bridge_rpc:
    ip: consensus.lunaroasis.net
    rpc_port: 26657
    grpc_port: 9090
```

Reply:
in env.yml ?

Reply:
yes